packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -110,10 +110,14 @@ const Checkbox = ({
 
   const checkboxClasses = cx([
     'checkbox',
-    { 'checkbox-inline': isInline },
+    { 'is-inline': isInline },
     customProps.className,
   ]);
 
+  const selectContainer = cx([
```

Comment:
Ok, so I was wrong to remove this. If a theme uses a css checkbox, removing `select-container` and adding the `padding-top` &`padding-bottom` to `label` pushes the label text to be misaligned (because the padding applies to the label text, but not the absolutely positioned css checkbox). So having this outer container to increase the hit-box should be fine.